client/my-sites/themes/is-full-site-editing-theme.ts

```diff
@@ -1,4 +1,6 @@
-export function isFullSiteEditingTheme( theme ) {
+import { Theme } from 'calypso/types';
+
+export function isFullSiteEditingTheme( theme: Theme | null ): boolean | undefined {
```

Comment:
I think we can always return a `boolean` because if we don't have any theme features, we are safe to assume that `block-templates` is not there, and so the theme is non-FSE.

```suggestion
export function isFullSiteEditingTheme( theme: Theme | null ): boolean {
```

Reply:
Sounds good 👍